sdk/identity/identity/review/identity.api.md

```diff
@@ -66,12 +70,26 @@ export enum AzureAuthorityHosts {
 
 // @public
 export class AzureCliCredential implements TokenCredential {
+    constructor(options?: AzureCliCredentialOptions);
     getToken(scopes: string | string[], options?: GetTokenOptions): Promise<AccessToken>;
+    }
+
+// @public
+export interface AzureCliCredentialOptions extends TokenCredentialOptions {
+    allowMultiTenantAuthentication?: boolean;
```

Comment:
This should go on GetTokenOptions also.

Reply:
Oh ok ok! Moving this to GetTokenOptions

Reply:
This shouldn't go on `GetTokenOptions` but rather should be at to `TokenCredentialOptions` as it's something users configure when they create the credential.

Reply:
Oh wow. Well, seems I might be able to do the update by undoing some of my changes. Thank you for the feedback!